.asf.yaml

```diff
@@ -27,7 +27,7 @@ github:
     # Enable squash button
     squash: true
     # Disable merge button
-    merge: true
+    merge: false
```

Comment:
What is the purpose of modifying this parameter?

Reply:
Looks some commits from develop branch are merged.

Reply:
yeah, this submission is the result of rebase the develop branch.